src/pages/registration/FileList.tsx

```diff
@@ -82,63 +84,18 @@ export const FileList = ({ title, files, uppy, remove, baseFieldName }: FileList
-                                  components={[<Box key="1" component="span" sx={{ fontWeight: 'bold' }} />]}
-                                />
-                              </Typography>
-                            </>
-                          ) : (
-                            <>
-                              <Trans
-                                i18nKey="registration.files_and_license.version_helper_text"
-                                components={[<Typography sx={{ mb: '1rem' }} key="1" />]}
-                                values={{ buttonText: t('my_page.messages.get_curator_support') }}
-                              />
-                              <Typography sx={{ mb: '1rem' }}>
-                                <Trans
-                                  i18nKey="registration.files_and_license.version_accepted_helper_text"
-                                  components={[<Box key="1" component="span" sx={{ fontWeight: 'bold' }} />]}
-                                />
-                              </Typography>
-                              <Typography sx={{ mb: '1rem' }}>
-                                <Trans
-                                  i18nKey="registration.files_and_license.version_published_helper_text"
-                                  components={[<Box key="1" component="span" sx={{ fontWeight: 'bold' }} />]}
-                                />
-                              </Typography>
-                              <Typography sx={{ mb: '1rem' }}>
-                                <Trans
-                                  i18nKey="registration.files_and_license.version_publishing_agreement_helper_text"
-                                  components={[<Box key="1" component="span" sx={{ fontWeight: 'bold' }} />]}
-                                />
-                              </Typography>
-                              <Typography sx={{ mb: '1rem' }}>
-                                <Trans
-                                  i18nKey="registration.files_and_license.version_embargo_helper_text"
-                                  components={[<Box key="1" component="span" sx={{ fontWeight: 'bold' }} />]}
-                                />
-                              </Typography>
-                            </>
-                          )}
+                          <Trans
+                            i18nKey="registration.files_and_license.version_helper_text"
+                            components={translationComponents}
+                          />
+                          <Trans
+                            i18nKey="registration.files_and_license.version_accepted_helper_text"
+                            components={translationComponents}
+                          />
+                          <Trans
+                            i18nKey="registration.files_and_license.version_published_helper_text"
+                            components={translationComponents}
+                          />
```

Comment:
Sånn det er satt opp nå hvor oversettelsen til hver `Trans` selv holder styr på `<p>`, `<heading>` osv så virker det litt rart at det er delt opp i 3 ulike keys, og at man fortsatt må justere 3 keys for å endre ting her